### DISCUSSION TITLE
Help: "The translations in several languages have failing checks" -- but it does not or I don't see any

### DISCUSSION THREAD

**Discussion by pmamat** [WeblateOrg/weblate#9234] [Q&A] [answered]
Hello Weblate Community, thanks for this great tool!

For many strings I get the message "The translations in several languages have failing checks", but I don't see failing checks.

![weblate-1-overview](https://github.com/WeblateOrg/weblate/assets/294254/79bd7be0-375e-4cb1-93af-1dd0d85783b7)

![weblate-3-detail](https://github.com/WeblateOrg/weblate/assets/294254/dcc97200-6033-4364-9e15-a4a8141e8869)

I thought that this warning occured, because there were languages which we did not translate yet. Since they now, those langugages aren't required any more, I deleted those languages. Could this be a caching issue? But I failed removing the issue by simply saving the string again...

Any thoughts or ideas why this is happening?

Thanks in advance for all your help!

Kind regards,
 Philipp

PS: The installed Weblate version is 4.15

**Comment by nijel** [comment, marked as answer]
The checks are not automatically updated after translation removal, fixed in ea18d1d4d5, see #9250.

**Reply by pmamat** [reply]
Thanks a lot for your help!
